src/com/xilinx/rapidwright/rwroute/RWRouteConfig.java

```diff
@@ -94,6 +94,16 @@ public class RWRouteConfig {
     private boolean lutRoutethru;
     /* true to enable GND -> VCC optimization for LUT inputs */
     private boolean invertGndToVccForLutInputs;
+    /* true to use HUS */
+    private boolean useHUS;
+    /* The value of alpha in HUS*/
+    private float HUSalpha;
+    /* The value of Beta in HUS*/
+    private float HUSbeta;
+    /* The threshold for determining whether a case is a congested design in HUS */
+    private float HUSinitialCongestedThreshold;
+    /* The threshold for determining whether to start using historical-centric updating in HUS */
+    private float HUSactivateThreshold;
```

Comment:
`HUSalpha`, `HUSbeta`, `HUSinitialCongestedThreshold` and `HUSactivateThreshold` should also be refactored to avoid starting with a capital letter.  Another option is to begin with an underscore (`_`).

Reply:
OK, I've refactored these variables.